code/modules/mob/mob.dm

```diff
@@ -151,7 +151,7 @@
 			if(target.see_invisible<invisibility) //if src is invisible to us,
 				msg = blind_message
 			//the light object is dark and not invisible to us, darkness does not matter if you're directly next to the target
-			else if(T.lighting_object && T.lighting_object.invisibility <= target.see_invisible && T.is_softly_lit() && !in_range(T,target))
+			else if(target.lighting_alpha > LIGHTING_PLANE_ALPHA_MOSTLY_INVISIBLE && T.is_softly_lit() && !in_range(T,target))
```

Comment:
This and the subsequent similar line is very problematic, I'd rather not mess with visible_message(), but KylerAce removed the invisibility var from lighting objects, and I'd rather not try to add it back in. This line is the same as on tg before this PR. However, I'm still not sure if this is the correct way to handle this.

Reply:
See also https://github.com/tgstation/tgstation/pull/58991#discussion_r650454567